vertx-web/src/main/java/io/vertx/ext/web/sstore/impl/SessionImpl.java

```diff
@@ -162,7 +219,7 @@ private Buffer writeDataToBuffer() {
       Buffer buffer = Buffer.buffer();
       buffer.appendInt(data.size());
```

Comment:
If the session is destroyed then it is not stored. The method is not called. The call is managed by the store.